### ISSUE TITLE
Netlify logo

### ISSUE THREAD

**Issue by janosh** [jacobwgillespie/styled-icons#1263]
Any icon pack around that includes the Netlify logo?

Btw, awesome work you're doing on this package @jacobwgillespie! 🎉

**Comment by jacobwgillespie** [comment]
It does appear that Netlify is missing from all existing packs!

I haven't had much time to fix the SVG issue that's blocking the latest release, but once I get around to that, it looks like I can add either `simple-icons` and/or `core-ui` icons to pick up a lot of brand icons, including Netlify.

**Comment by jacobwgillespie** [comment]
@janosh this has released, there's a Netlify icon now 🎉 

https://styled-icons.js.org/?s=netlify

**Comment by janosh** [comment]
Thanks for working on this! Just tried it out. I'm getting the error

> Can't resolve 'styled-icons/simple-icons'

from this line:

```js
import { Gatsby, Github, Netlify } from 'styled-icons/simple-icons'
```

**Comment by jacobwgillespie** [comment]
Ah, good catch, this will be fixed by #1275. Also for reference, the packs are installable individually, in case you'd like to slim your `node_modules`. 🙂 For instance:

```diff
- import { Gatsby, Github, Netlify } from 'styled-icons/simple-icons'
+ import { Gatsby, Github, Netlify } from '@styled-icons/simple-icons'
```

**Comment by janosh** [comment]
> Ah, good catch, this will be fixed by #1275. Also for reference, the packs are installable individually, in case you'd like to slim your `node_modules`. 🙂 For instance:
> 
> ```diff
> - import { Gatsby, Github, Netlify } from 'styled-icons/simple-icons'
> + import { Gatsby, Github, Netlify } from '@styled-icons/simple-icons'
> ```

I know. It's just that I use many of the packs and I'm too lazy to install them individually. 😄

**Comment by jacobwgillespie** [comment]
Totally, it's why the `mono-package` still exists. 🙂 

This should be fixed in v10.4.1: https://unpkg.com/browse/styled-icons@10.4.1/simple-icons/

**Comment by janosh** [comment]
Just installed v10.4.1. Still getting the same error unfortunately. 😞

**Comment by jacobwgillespie** [comment]
@janosh yep, my bad, `styled-icons` was missing the new dependency for `simple-icons` - this has been fixed in v10.4.2. I am able to import the Netlify icon successfully now from a clean CRA app.

**Comment by janosh** [comment]
@jacobwgillespie Same here, now it works! 🎉
